temporalcli/commandsgen/commands.yml

```
@@ -374,6 +375,99 @@ commands:
+        * if there is no retry left - the activity will be completed.
+        * if there are more retries left - the activity will be paused.
+      
+      Specify the
+      Activity and Workflow IDs:
+
+      ```
+      temporal activity pause \
+          --activity-id YourActivityId \
+          --workflow-id YourWorkflowId
+      ```
+    options:
+      - name: activity-id
+        short: a
+        type: string
+        description: Activity ID to pause.
+        required: true
+      - name: identity
+        type: string
+        description: Identity of the user submitting this request.
+    option-sets:
+      - workflow reference
+
+  - name: temporal activity unpause
+    summary: Unpause an Activity
+    description: |
+      Unpauses the execution of an activity specified by its ID.
+      Returns a `NotFound` error if there is no pending activity 
+      with the provided ID.
+      There are two 'modes' of unpausing an activity:
+        'resume' - Default. If the activity is paused, it will be resumed 
+        and scheduled for execution.
+        * If the activity is currently running Unpause with 'resume' has 
+          no effect.
+        * if 'no_wait' flag is set and the activity is waiting, the activity 
+          will be scheduled immediately.
+        'reset' - If the activity is paused, it will be reset to its initial 
+        state and (depending on parameters) scheduled for execution.
+        * If the activity is currently running, Unpause with 'reset' will reset 
+          the number of attempts.
+        * if 'no_wait' flag is set, the activity will be scheduled immediately.
+        * if 'reset_heartbeats' flag is set, the activity heartbeat timer 
+          and heartbeats will be reset.
+      If the activity is in waiting for retry and past it retry timeout, 
+      it will be scheduled immediately.
+      Once the activity is unpaused, all timeout timers will be regenerated.
+
+      Specify the
+      Activity and Workflow IDs:
+
+      ```
+      temporal activity unpause \
+          --activity-id YourActivityId \
+          --workflow-id YourWorkflowId
+          --reset
+          --no-wait
+          --reset-heartbeats
+      ```
+    options:
+      - name: activity-id
+        short: a
+        type: string
+        description: Activity ID to pause.
+        required: true
+      - name: identity
+        type: string
+        description: Identity of the user submitting this request.
+      - name: reset
+        type: bool
+        description: Also reset the activity.
+      - name: no-wait
+        type: bool
+        description: |
+          If reset flag is provided, the activity will be scheduled immediately.
+          If reset flag is not provided, the activity will be scheduled if 
+          it is waiting for retry.
+      - name: reset-heartbeats
+        type: bool
+        description: I reset flag is provided - reset the activity heartbeats.
```

```suggestion
        description: If reset flag is provided - reset the activity heartbeats.
```

Also, after merge, may want to see if docs team wants to do a pass on these docs (they have cleaned up other CLI docs for us)

---

```suggestion
        description: Reset the Activity's heartbeat timeout. Only works with --reset.
```